tmcprototype/CspSubarray/CspSubarray/CspSubarray.py

```diff
@@ -127,6 +128,8 @@ def init_device(self):
         self._receptor = []
         self._correlations = " "
         self._health_state = 0
```

Comment:
self._health_state = 0
self._obs_state = 1
self._opstate = 2
Not clear which state is represented by '2'. Use of enum recommended. Else add comment for better understanding.

Reply:
Nice suggestion, this feature is available in PyTango 9.2.5+. We need to test it before implementing throughout the project. It was implemented as part of [this](https://github.com/tango-controls/pytango/pull/221) pull request in PyTango.

Reply:
1. Resoved these comments in commit [aaf968f95e7ba3fb5e6aed85ed5e3ef9ff6bc2cf].
2. Added comments on CspSubarray.
3. Added ENUMs on CspMasterLeafNode.

Reply:
Updated the SubarrayNode and CentralNode accordingly. The commit id is: [0938403d20f73f4fdc2a112cbd901b51afca5fe3](https://github.com/ska-telescope/tmc-prototype/commit/0938403d20f73f4fdc2a112cbd901b51afca5fe3)

Reply:
Resolved comments for DishLeafNode and SdpSubarrayLeafNode in commit  ba85d10655f89420768871fc02d63fd651de5e55